decam/makeCrosstalkDecam.py

```diff
@@ -0,0 +1,159 @@
+#!/usr/bin/env python3
+"""Convert a DECam crosstalk text table into LSST CrosstalkCalibs.
+"""
+import argparse
+import numpy as np
+import os.path
+import sys
+
+import lsst.ip.isr as ipIsr
+from lsst.daf.base import PropertyList
+from lsst.obs.decam import DecamMapper
+
+
+def makeDetectorCrosstalk(dataDict, force=False):
+    """Generate and write CrosstalkCalib from dictionary.
+
+    Parameters
+    ----------
+    dataDict : `dict`
+        Dictionary from ``readFile`` containing crosstalk definition.
+    """
+    dataDict['coeffs'] = dataDict['coeffs'].transpose()
+
+    decamCT = ipIsr.crosstalk.CrosstalkCalib.fromDict(dataDict)
+    # Lie to ingestCuratedCalibs, because it interprets calibDate as validStart.
```

Comment:
The conversion of existing defects wrote the yaml representation (what this script is attempting to duplicate) to files containing the explicit start points, and set CALIBDATE to that value.  The new defects use the midpoint of the input observations (which I don't think it correct, either).  I've updated the comment to be more informative, and this will likely be revisited in the future when all the calibrations are more unified.